.github/workflows/ci.yml

```diff
@@ -9,7 +9,10 @@ on:
 jobs:
   test:
     name: Lint and test
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
@simoneb my knowledge of GH Actions is very limited. Is this the right approach to test both on Ubuntu and on Win?